src/analyze/analyze-security.c

```diff
@@ -1732,7 +1733,13 @@ static int assess(const SecurityInfo *info,
         int r;
 
         if (!FLAGS_SET(flags, ANALYZE_SECURITY_SHORT)) {
-                details_table = table_new(" ", "name", "description", "weight", "badness", "range", "exposure");
+                const char *first_column_name = " ";
+
+                /* The json output of the table should have a proper key name for the first field */
+                if (json_format_flags != JSON_FORMAT_OFF)
+                        first_column_name = "set";
+
+                details_table = table_new(first_column_name, "name", "description", "weight", "badness", "range", "exposure");
```

Comment:
I think in JSON mode the "name" column should get the assesor's `json_field` field instead of `id`. i.e. it's important the policy JSON and the output you generate here can be reasonably mapped to each other.

Reply:
This was addressed on lines 1808 and 1809 of the analyze-security.c file